StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethodFormFactory/SavedPaymentMethodFormFactory+Card.swift

```diff
@@ -73,14 +74,78 @@ extension SavedPaymentMethodFormFactory {
             return TextFieldElement.CensoredCVCConfiguration(brand: configuration.paymentMethod.card?.preferredDisplayBrand ?? .unknown).makeElement(theme: configuration.appearance.asElementsTheme)
         }()
 
+        let billingAddressSection: PaymentMethodElementWrapper<AddressSectionElement>? = {
+            guard configuration.canUpdate else {
+                return nil
+            }
+            switch configuration.billingDetailsCollectionConfiguration.address {
+            case .automatic:
+                return makeBillingAddressSection(configuration, collectionMode: .countryAndPostal(), countries: nil)
+            case .full:
+                return makeBillingAddressSection(configuration, collectionMode: .all(), countries: nil)
+            case .never:
+                return nil
+            }
+        }()
+
         let cardSection: SectionElement = {
             let allSubElements: [Element?] = [
                 panElement,
                 SectionElement.HiddenElement(cardBrandDropDown),
                 SectionElement.MultiElementRow([expiryDateElement, cvcElement]),
             ]
-            return SectionElement(elements: allSubElements.compactMap { $0 }, theme: configuration.appearance.asElementsTheme)
+            return SectionElement(title: billingAddressSection != nil ? String.Localized.card_information : nil,
+                                  elements: allSubElements.compactMap { $0 },
+                                  theme: configuration.appearance.asElementsTheme)
         }()
-        return cardSection
+        return FormElement(elements: [cardSection, billingAddressSection], theme: configuration.appearance.asElementsTheme)
+    }
+
+    func makeBillingAddressSection(
```

Comment:
Is there a clear way to reuse/DRY this up with what we have in PaymentSheetFormFactory? We have a function that nearly does the same.